src/node/index.ts

```diff
@@ -51,7 +51,10 @@ class BeaconNode {
 
     this.db = new LevelDB(this.conf.db);
     this.network = new P2PNetwork(this.conf.network);
-    this.eth1 = new EthersEth1Notifier(this.conf.eth1);
+    this.eth1 = new EthersEth1Notifier({
+      ...this.conf.eth1,
```

Comment:
The other modules currently have a constructor interface of `(opts, modules)`.
I'm not sure what the best option is, just FYI

Reply:
Oh yes, I will adjust it like other modules